### ISSUE TITLE
Change formatting so that the User's messages are aligned to the right-side of the conversation (parity with Android)

### ISSUE THREAD

**Issue by rpinto73** [nextcloud/spreed-ios#698]
In most messaging apps, the person who the phone belongs to (let's call them the User) has their messages appearing on the left-hand side, and the messages from the other people in the chat on the right-hand side. In the iOS Talk app, all the messages are left-aligned, which makes it harder to quickly visualize when the User is talking in the conversation. 

In the Android Nextcloud Talk App, the User's messages appear on the right-hand side in a blue block with the other person's messages on the left-hand side in a black block. I'd like the iOS version to achieve parity with this feature by making conversations appear the same way in iOS as they do in Android.
Thanks.

**Comment by Dennis1993** [comment]
Hey,

we want to use Nextcloud Talk in our company and test it since 2 weeks. On friday we had a feedback round with all testers.

All testers said this one which is discribed here.

The messages should be displayed in bubbles (like WhatsApp) and the own messages should be aligned on the right hand. So the overview is more better than the current left aligned view.

I hope this can be integrated in the next version to start with it. ☺️

Thanks!!!

**Comment by SystemKeeper** [comment]
> I hope this can be integrated in the next version to start with it. ☺️

I'm not involved in the roadmap, but personally I don't think this will happen in the next version (if it happens). Judging from the current codebase this would require some work to get this done nicely.
The current implementation is obviously based around the idea to be as close as possible to it's web counterpart and it does a pretty good job doing so. In my opinion this is a good thing.

**Comment by MagicTrevor** [comment]
Would definitely like to see this as well. Or at least the ability (if possible) to toggle it.

**Comment by AndyXheli** [comment]
Yes, I would also agree id also love to see this especially since android talk has it. It would be nice to have the same user experience on both platforms

**Comment by thijsp** [comment]
This would indeed increase the readability of messages. 
I would like to add that it would also be nice to be able to have alternating colours between the different members of the conversation. Or at least a different color for yourself. Like in the android app and many other messaging applications.

**Comment by Dennis1993** [comment]
Is any progress on this nice idea to optimize the overview for iOS? ☺️
In Android it is available. On the website the feature is shown on the front page but for iOS Users this feature is not useable 🫣
May a timeline for that?
Thank you so much!

**Comment by rajmondburgaj** [comment]
Is there any concrete plan to implement this? Without this feature we are having difficulty to use the app altogether.

**Comment by Dennis1993** [comment]
We are waiting too, because the overview on iOS is very bad. All apps are using the bubble style (Signal, WhatsApp...) and the reading of text is hard without it. On nextcloud.com is this in the screenshot presented but it is fake (for iOS) :D

![image](https://github.com/nextcloud/talk-ios/assets/1473674/d560728b-9c56-4521-b134-c552755f056c)

We are testing this app for our company (because privacy) and many users said that the overview on iOS is bad.
So I hope it can be integrated on iOS too.
**THANKS!**

**Comment by lotofbits** [comment]
![grafik](https://github.com/nextcloud/talk-ios/assets/217380/95589fd2-46de-4439-a860-e06402cdd30e)

Would be really great, if this fresh design shown on nextcloud.com could be applied to iOS =)

**Comment by Dennis1993** [comment]
Any news here? 🙈

**Comment by SystemKeeper** [comment]
There are no news at the moment, sorry. If there’s anything to tell, we will update the issue. In case you like to see this feature, please leave a +1 in the first post. Everything else will not move this issue forward

**Comment by SystemKeeper** [comment]
This is done with https://github.com/nextcloud/talk-ios/pull/2153

**Comment by SystemKeeper** [comment]
It’s available in our latest TestFlight: feel free to test https://testflight.apple.com/join/cxzyr1eO
